doc/user/user/project/index.md

```diff
@@ -310,6 +310,7 @@ The following data will be synchronized:
 * Basic project metadata
 * FASTQ sequencing data associated with samples
 * Assemblies associated with samples (including both assemblies created by pipelines within IRIDA, and uploaded assemblies)
+* FAST5 sequencing data associated with samples
```

Comment:
Remove the fast5 section below in the **will not** section.

Reply:
Removed in ccf8d68de322f55c99d3283986c593dd49df391e